README.md

```diff
@@ -90,6 +91,161 @@ This WordPress project is intended to provide:
   define( 'SCRIPT_DEBUG', true );
   ```
 
+## Automated Testing
```

Comment:
Ok, I'll transfer the Troubleshooting section to the Wiki. I'll also group together a simplified version  of automated testing and send the rest to the wiki.